packages/ios_platform_images/example/android/.gitignore

```diff
@@ -0,0 +1,7 @@
+gradle-wrapper.jar
```

Comment:
(optional) @amirh, do we support plugins with no android implementation and/or example currently?